gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXRepo.cs

```diff
@@ -20,8 +20,8 @@ public static class GTXRepo
         /// <param name="repoPath">the repository main path</param>
         public static void Init(string repoPath)
         {
-            if (!Repository.IsValid(repoPath))
-                Repository.Init(repoPath, false);
+            if (!Repository.IsValid(repoPath))            
+             Global.error(repoPath  + " is not a valid git repo");            
```

Comment:
Let me know if you need any help. I could easily test things out. Any idea of level of effort required?